README.md

```diff
@@ -6,6 +6,8 @@ This page contains a list of example codes written with Optuna.
 ### Simple Black-box Optimization
 
 * [Quadratic function](./quadratic_simple.py)
+- [Quadratic multi-objective function](./multi_objective/quadratic_simple.py)
+- [Quadratic function with constraints](./quadratic_simple_constraint.py)
```

Comment:
Seems like `*` should be used for listing

Reply:
Nice catch!  if necessary, I'm happy to include this too in https://github.com/optuna/optuna-examples/pull/217 as a followup.

Reply:
Thanks for the comment! We appreciate your follow-up.

Reply:
Thank you for your response; I've updated the PR.